src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
```

Comment:
Since both fields are required, there is no need to use empty constructor. You can directly return Ball instance with _color_ and _number_ fields from the method